records/generator.go

```diff
@@ -25,6 +26,8 @@ import (
 // Map host/service name to DNS answer
 // REFACTOR - when discoveryinfo is integrated
 // Will likely become map[string][]discoveryinfo
+// Effectively we're (ab)using the map type as a set
+// It used to have the type: rrs map[string][]string
```

Comment:
I thought the values were converted to a set for higher efficiency during building? Is it that much overhead to transform it here?